synapse/models/economic.py

```diff
@@ -61,10 +67,29 @@ def getModelDefs(self):
                 ('econ:fin:tick', ('guid', {}), {
                     'doc': 'A sample of the price of a security at a single moment in time'}),
 
-                # econ:acct:bill
-                # econ:bank:us:aba:rtn ( ABA Routing Number )
-                # econ:bank:us:account = (econ:bank:us:aba:rtn, acct)
-                # econ:bank:swift:...
+                ('econ:bank:account:type:taxonomy', ('taxonomy', {}), {
+                    'doc': 'A bank account type taxonomy.'}),
+
+                ('econ:bank:account', ('guid', {}), {
```

Comment:
for consistency with the existing econ:acct:* forms, do we want to name this econ:bank:acct?

Reply:
No, the abbreviation is not worth the confusion. The up coming service model updates will also use `inet:service:account` :+1: